### PR TITLE
Filter sold and reserved estates from similar estates.

### PR DESCRIPTION
**Pull request by FD-Websolutions**

### CODE REVIEW

File: plugin/Filter/DefaultFilterBuilderSimilarEstates.php [hunk 1]
```diff
@@ -86,6 +86,9 @@ public function buildFilter(): array
 			$filter['Id'] []= ['op' => 'not in', 'val' => $this->_excludeIds];
 		}
 
+		$filter['verkauft'] []= ['op' => 'not in', 'val' => 1];
+		$filter['reserviert'] []= ['op' => 'not in', 'val' => 1];
```
Comment (jayay) [inline plugin/Filter/DefaultFilterBuilderSimilarEstates.php:90]: Why not insert it at the top? (line 63)

**Review by jayay — CHANGES_REQUESTED** [review]
I don't know if you've talked to Pawel about this, but it makes sense to me. Would you mind removing the `.DS_Store` files, please?

File: plugin/Filter/DefaultFilterBuilderSimilarEstates.php (hunk 1 above)
Comment (FD-Websolutions) [reply plugin/Filter/DefaultFilterBuilderSimilarEstates.php]: I changed the position.

**Review by jayay — APPROVED** [review]

**Comment by jayay** [comment]
Released in v2.8.3.

**Pull request merged**